san/tests/test_get.py

```diff
@@ -79,6 +93,385 @@ def test_error_response(mock):
             interval="1d"
         )
 
+
+@patch('san.graphql.requests.post')
+def test_get_without_transform(mock):
+    expected = {
+        'query_0':
+            [{'datetime': '2018-06-01T00:00:00Z', 'activeAddresses': 2},
+             {'datetime': '2018-06-02T00:00:00Z', 'activeAddresses': 4},
+             {'datetime': '2018-06-03T00:00:00Z', 'activeAddresses': 6},
+             {'datetime': '2018-06-04T00:00:00Z', 'activeAddresses': 6},
+             {'datetime': '2018-06-05T00:00:00Z', 'activeAddresses': 14}]
+    }
+
+    # The value is passed by refference, that's why deepcopy is used for
+    # expected
+    mock.return_value = TestResponse(status_code=200, data=deepcopy(expected))
+
+    res = san.get(
+        "daily_active_addresses/santiment",
+        from_date="2018-06-01",
+        to_date="2018-06-05",
+        interval="1d"
+    )
+    expected = convert_to_datetime_idx_df(expected['query_0'])
```

Comment:
`expected_df` - reusing var names is confusing, even more when it also changes the type (dict -> pandas data frame)

Reply:
Just did that, it really does confuse